Automattic/wp-calypso
# /plugins: Integrate the LOHP navigation bar

#### Proposed Changes

* /plugins: Integrate the LOHP navigation bar
* 1000-gh-Automattic/martech

#### Testing Instructions

* Log out
* Navigate to /plugins
* Ensure that you see the sitewide header Nav bars.
* The page should be unaffected if logged in

#### Pre-merge Checklist






Related to #
